### PR TITLE
Created api that demonstrates symfony/http-foundation possibilities

### PR DESCRIPTION
**Pull request by L0PE**

### CODE REVIEW

File: src/http/Api/rundomSrarusCode.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
```
Comment (Olkoson) [inline src/http/Api/rundomSrarusCode.php]: Please, fix a file name.

File: src/http/Api/setSatusCode.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?php
```
Comment (Olkoson) [inline src/http/Api/setSatusCode.php]: weird file name for this functionality.
Better use notFoundCode or something like that.

File: src/http/Api/rundomSrarusCode.php (hunk 1 above)
Comment (L0PE) [reply src/http/Api/rundomSrarusCode.php]: You`re right

File: src/http/Api/setSatusCode.php (hunk 2 above)
Comment (L0PE) [reply src/http/Api/setSatusCode.php]: Done

**Pull request merged**